staff/joseluis-juste/CSS/A2/css/styles.css

```diff
@@ -0,0 +1,217 @@
+    color: white;
+    border-radius: 22px; }
+  .group2 div:first-child a:nth-child(6) {
+    background-color: #ED5463;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    color: white;
+    border-radius: 22px; }
+.group2 div:last-of-type {
+  -webkit-box-pack: justify;
+      -ms-flex-pack: justify;
+          justify-content: space-between;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex; }
+  .group2 div:last-of-type a:nth-child(1) {
+    border: 1px solid rgba(181, 171, 172, 0.7);
+    color: rgba(181, 171, 172, 0.7);
+    border-radius: 22px; }
+  .group2 div:last-of-type a:nth-child(2) {
+    border: 1px solid #5C9DED;
+    color: #5C9DED;
+    border-radius: 22px; }
+  .group2 div:last-of-type a:nth-child(3) {
+    border: 1px solid #42CB6E;
+    color: #42CB6E;
+    border-radius: 22px; }
+  .group2 div:last-of-type a:nth-child(4) {
+    border: 1px solid #4EC2E8;
+    color: #4EC2E8;
+    border-radius: 22px; }
+  .group2 div:last-of-type a:nth-child(5) {
+    border: 1px solid #FCCE54;
+    color: #FCCE54;
+    border-radius: 22px; }
+  .group2 div:last-of-type a:nth-child(6) {
+    border: 1px solid #ED5463;
+    color: #ED5463;
+    border-radius: 22px; }
+
+.group3 div:first-child {
+  -webkit-box-pack: start;
+      -ms-flex-pack: start;
+          justify-content: start;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex; }
+  .group3 div:first-child a:nth-child(1) {
+    background-color: #EAEAEA;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    color: #000000;
+    border-radius: 22px; }
+  .group3 div:first-child a:nth-child(2) {
+    background-color: #5C9DED;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    color: white;
+    border-radius: 22px; }
+  .group3 div:first-child a:nth-child(3) {
+    background-color: #42CB6E;
+    border: 1px solid rgba(0, 0, 0, 0.1);
+    color: white;
+    border-radius: 22px; }
+  .group3 div:first-child a {
+    width: 85px;
+    line-height: 25px;
+    height: 29px; }
+  .group3 div:first-child a:nth-child(n + 2) {
+    margin-left: 15px; }
+
+.group4 div {
+  -webkit-box-pack: start;
+      -ms-flex-pack: start;
+          justify-content: start;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex; }
+  .group4 div a:nth-child(1) {
+    border: 1px solid #4EC2E8;
+    color: white;
+    border-radius: 15px;
+    background-color: #4EC2E8;
+    font-size: 2em;
+    width: 144px;
+    height: 67px;
+    line-height: 61px; }
+  .group4 div a:nth-child(2) {
+    border: 1px solid #FCCE54;
+    color: white;
+    border-radius: 15px;
+    background-color: #FCCE54;
+    font-size: 2em;
+    width: 144px;
+    height: 67px;
+    line-height: 61px; }
+  .group4 div a:nth-child(3) {
+    border: 1px solid #ED5463;
+    color: white;
+    border-radius: 15px;
+    background-color: #ED5463;
+    font-size: 2em;
+    width: 144px;
+    height: 67px;
+    line-height: 61px; }
+  .group4 div a:nth-child(n + 2) {
+    margin-left: 15px; }
```

Comment:
This code is difficult to read, maybe add more spaces betheen div subgroup?

Reply:
You have right but it has an explanation. I have used SASS (https://sass-lang.com/) so when you uses SASS, you can program your css in files of kind scss. Then these files are compiled to a CSS file, for that reason you see all code very collapsed because Iam not programing in CSS directly. If you want to see the code in a better way, please, open de SASS folder